Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Remote_PsExec_with_LOLBIN_command_execution_alert.yml

```diff
@@ -0,0 +1,1029 @@
+    type: condition
+    task:
+      id: b07b3289-4878-4263-8d73-ffc488a3389a
+      version: -1
+      name: Found additional alerts or suspicious command line?
+      description: Check if a malicious verdict was found.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "22"
+      Malicious:
+      - "15"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: CommandlineVerdict
+            iscontext: true
+          right:
+            value: {}
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXDR.Incident
+                accessor: critical_severity_alert_count
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.CriticalAlertsThreshold
+            iscontext: true
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.Incident.high_severity_alert_count
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.HighAlertsThreshold
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: f7f4ff17-5b9a-4586-8d4c-40c135b8ad4f
+    type: playbook
+    task:
+      id: f7f4ff17-5b9a-4586-8d4c-40c135b8ad4f
+      version: -1
+      name: Threat Hunting - Generic
+      description: "This playbook enables threat hunting for IOCs in your enterprise. It currently supports the following integrations: \n- Splunk\n- Qradar\n- Pan-os \n- Cortex data lake \n- Autofocus\n- Microsoft 365 Defender"
+      playbookName: Threat Hunting - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      IPAddress:
+        complex:
+          root: inputs.SrcIPAddress
+          transformers:
+          - operator: uniq
+      MD5:
+        complex:
+          root: incident.xdralerts
+          filters:
+          - - operator: in
+              left:
+                value:
+                  simple: incident.xdralerts.alert_id
+                iscontext: true
+              right:
+                value:
+                  simple: inputs.alerts_ids
+                iscontext: true
+              ignorecase: true
+          accessor: causalityactorprocessimagemd5
+          transformers:
+          - operator: uniq
+      QRadarTimeFrame:
+        simple: LAST 7 DAYS
```

Comment:
I see the default is 7 days. Do we want to keep it 7 days? Because the other playbook 'Cortex XDR - Endpoint Investigation' checks the last 2 hours